modules/sops/default.nix

```diff
@@ -320,7 +320,7 @@ in {
         }]) cfg.secrets)
       );
 
-      sops.environment.SOPS_GPG_EXEC = lib.mkIf (cfg.gnupg.home != null || cfg.gnupg.sshKeyPaths != []) (lib.mkDefault "${pkgs.gnupg}/bin/gpg");
+      sops.environment.SOPS_GPG_EXEC = lib.mkIf (cfg.gnupg.home != null || cfg.gnupg.sshKeyPaths != []) (lib.mkDefault (lib.getExe (pkgs.gnupg.override { enableMinimal = true; })));
```

Comment:
Do we cache the minimal build in hydra?

Reply:
Kinda https://github.com/NixOS/nixpkgs/blob/e4c22dc16b36c17bea471ecc3ae616f64e69d3d2/pkgs/top-level/all-packages.nix#L27888-L27891, should I do a PR to promote that to a top-level attr?